packages/office-ui-fabric-react/src/components/Shimmer/Shimmer.types.ts

```diff
@@ -169,3 +216,26 @@ export enum ShimmerElementsDefaultHeights {
    */
   circle = 24
 }
+
+/**
+ * Interface describing the possible color customizations of Shimmer.
+ */
+export interface IShimmerColors {
```

Comment:
I was a bit confused by the naming, esp. between `shimmerMain` and `surroundingSpace`. How does this set look:

- `backgroundColor` (instead of `surroundingSpace`)
- `shimmerColor` (instead of `shimmerMain`)
- `shimmerWaveColor` / `shimmerAltColor` / `shimmerSecondaryColor`  (instead of `shimmerWave`)

We could possibly drop `color` at the end as they are in `IShimmerColors` and keep it simple

Reply:
Completely agree on that... I was hesitating on the naming so thanks for the suggestion. Will make the change.